toolchain/toolchain.bzl

```diff
@@ -96,6 +101,41 @@ def arm_none_eabi_toolchain(name, gcc_tool = "gcc", target_compatible_with = [],
             toolchain_type = "@bazel_tools//tools/cpp:toolchain_type",
         )
 
-def register_arm_none_eabi_toolchain(name):
+def arm_none_eabi_toolchain(name, version = "13.2.1", copts = [], **kwargs):
+    """
+    Create an arm-none-eabi toolchain with the given configuration.
+
+    Args:
+        name: The name of the toolchain.
+        gcc_tool: The gcc tool to use. Defaults to "gcc". [gcc, c++, cpp]
+        target_compatible_with: A list of constraint values to apply to the toolchain.
+        copts: A list of compiler options to apply to the toolchain.
+        linkopts: A list of linker options to apply to the toolchain.
+        version: The version of the gcc toolchain.
+    """
+    _arm_gnu_toolchain(name, toolchain = "arm_none_eabi",
+                       toolchain_prefix = "arm-none-eabi", version = version,
+                       abi_version = "eabi", copts = ["-nostdinc"] + copts,
```

Comment:
nit: we just merged a change to optionally disable standard includes (which sometimes are needed for C++ folks), it would be great to use those changes. Happy to patch this otherwise excellent PR if you don't have the time